pxr/imaging/plugin/hdRpr/volume.cpp

```diff
@@ -154,17 +159,59 @@ openvdb::CoordBBox computeGrigOnBB(TPtr grid)
 
 
 template <class TPtr, class TValueOnIt>
-void fillGridOnWithDefaultColor(TPtr grid, VtVec4fArray & out_grid, VtArray<size_t> & out_idx, openvdb::Coord & out_gridOnBBSize)
+void fillGridOnWithDefaultAlbedo(TPtr grid, VtArray<float> & out_density_grid, VtArray<size_t> & out_density_idx, VtArray<float> & out_albedo_grid, VtArray<size_t> & out_albedo_idx, openvdb::Coord & out_gridOnBBSize)
 {
-	openvdb::CoordBBox gridOnBB = computeGrigOnBB< TPtr, TValueOnIt>(grid);
+	openvdb::CoordBBox gridOnBB = computeGridOnBB< TPtr, TValueOnIt>(grid);
 	out_gridOnBBSize = gridOnBB.max() - gridOnBB.min();
 
 	for (TValueOnIt iter = grid->beginValueOn(); iter; ++iter) {
-		out_grid.push_back(defaultColor);
-		out_idx.push_back(coordToIndex(iter.getCoord(), gridOnBB, out_gridOnBBSize));
+		out_density_grid.push_back(grid->getAccessor().GetValue(iter.getCoord()));
+		out_albedo_grid.push_back(defaultColor);
```

Comment:
`out_albedo_grid` - vector of floats, `defaultColor` - GfVec4f

Reply:
This function removed in latest commit.